vars/tasks.groovy

```diff
@@ -49,6 +49,20 @@ def xpackCiGroups() {
   tasks(ciGroups.collect { kibanaPipeline.xpackCiGroupProcess(it, true) })
 }
 
+def xpackCiGroupDocker() {
+  task {
+    workers.ci(name: 'xpack-cigroups-docker', size: 'm', ramDisk: true) {
+      kibanaPipeline.downloadDefaultBuildArtifacts()
+      kibanaPipeline.bash("""
+        mkdir -p ../kibana-build-xpack
```

Comment:
I think using `cd "${env.WORKSPACE}"` and removing the `..` from paths would make sense